lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -345,6 +346,7 @@ public class TaskQueryImpl implements TaskQuery {
     this.orderByInner = new ArrayList<>();
     this.filterByAccessIdIn = true;
     this.withoutAttachment = false;
+    this.lockResults = 0;
```

Comment:
I don't know if we maybe can solve this differently. For now we can keep it like this but we should look into this once we have more time again.